src/App.vue

```diff
@@ -19,12 +24,31 @@ export default {
     LTileLayer,
     LMarker,
     MainMap,
-    Sidebar
+    Sidebar,
+    PointDetails,
+    MyMarker
   },
   data() {
     return {
       currentView: "MainMap"
     };
+  },
+  created: async function() {
+    const response = await instance.get(process.env.VUE_APP_HOSTNAME + "points/", {
+      params: {}
+    });
+    const json = response.data.results;
+    const count = response.data.count;
+    json.forEach(point => this.$store.commit('pushPoints', point));
+
```

Comment:
For api calls you could use Vuex actions, then call mutation inside the action. Look for an example here: https://zendev.com/2018/05/21/vuex-perfect-interface-frontend-backend.html
Every api call should be executed inside of Vuex.

Reply:
Moved in 65f5b89

Reply:
I'd still be really thankful for a review!